test_runlike.py

```diff
@@ -152,3 +152,6 @@ def test_cap_add(self):
     def test_devices(self):
         self.expect_substr("--device /dev/null:/dev/null:r")
         self.expect_substr("--device /dev/null:/dev/null", 2)
+
+    def test_workdir(self):
+        self.expect_substr("--workdir=/workdir")
```

Comment:
I think it's important to add a negative test here as well (expect not to find `--workdir` in the printout if it hasn't been set).